custom-commands/hospitality/test/hospitality_skill.json

```diff
@@ -0,0 +1,68 @@
+{
+  "Tests": [
+    {
+      "FileName": "input-files\\Test_FallbackCommand_TextInput.json",
+      "SingleConnection": true,
+      "Skip": false,
+	  "IgnoreActivities": [
+        {
+          "type": "event",
+        }
+      ]
+    },
+	{
+      "FileName": "input-files\\Test_TurnOnOff_TextInput.json",
+      "SingleConnection": true,
+      "Skip": false,
+	  "IgnoreActivities": [
+        {
+          "type": "event",
+        }
+      ]
+    },
+	{
+      "FileName": "input-files\\Test_Help_TextInput.json",
+      "SingleConnection": true,
+      "Skip": false,
+	  "IgnoreActivities": [
+        {
+          "type": "event",
+        }
+      ]
+    },
+	{
+      "FileName": "input-files\\Test_Greeting_TextInput.json",
+      "SingleConnection": true,
+      "Skip": false,
+	  "IgnoreActivities": [
+        {
+          "type": "event",
+        }
+      ]
+    },
+	{
+      "FileName": "input-files\\Test_AdjustTemperature_TextInput.json",
+      "SingleConnection": true,
+      "Skip": false,
+	  "IgnoreActivities": [
+        {
+          "type": "event",
+        }
+      ]
+    },
+	{
+      "FileName": "input-files\\Test_OpenClose_TextInput.json",
+      "SingleConnection": true,
+      "Skip": false,
+	  "IgnoreActivities": [
+        {
+          "type": "event",
+        }
+      ]
+    }
+  ],
+  "SubscriptionKey": "fd0d3bd0b03f4e36a149e342b56d5004",
```

Comment:
Please replace these values with
Fake Speech Key: 89t7s45tyu2i7y9j908w345u57962eb2
Azure Region: westus2

Reply:
Use the fake values I have here: https://github.com/Azure-Samples/Cognitive-Services-Voice-Assistant/blob/master/clients/csharp-dotnet-core/voice-assistant-test/docs/json-templates/app-config-template.json

Reply:
Update hospitality_skill.json with suggested fake values for SubscriptionKey and CustomCommandsAppId.